documentation/src/main/asciidoc/chapter-3-defining-a-mapper.asciidoc

```diff
@@ -124,6 +124,50 @@ Collection-typed attributes with the same element type will be copied by creatin
 
 MapStruct takes all public properties of the source and target types into account. This includes properties declared on super-types.
 
+[[mapping-composition]]
+=== Mapping Composition (experimental)
+MapStruct supports the use of meta annotations. The `@Mapping` annotation can be defined with `@Target` `ElementType#ANNOTATION_TYPE`. This will allow `@Mapping` to be used on other (user defined) annotations for re-use purposes. For example:
```

Comment:
_can be defined_? Sounds like the user could do this.. but it actually _was defined with ..._? Following then in the next sentence: _will allow_ -> _allows_?

Reply:
true! I'll fix this.. thanks @chris922

Reply:
I rephrased it to: _The `@Mapping` annotation supports now `@Target` with `ElementType#ANNOTATION_TYPE` in addition to `ElementType#METHOD`. This allows `@Mapping` to be used ..._